python/lsst/ts/imsim/closedLoopTask.py

```diff
@@ -534,6 +548,10 @@ def _runSim(
             else:
                 self.ofcCalc.set_fwhm_data(fwhm, sensor_id)
 
+            # Flip zernikes that are not symmetric across the y-axis
+            # based upon flip in batoid coordinate system.
+            wfe[:, [1, 4, 6, 9, 11, 12, 14, 16]] *= -1.0
```

Comment:
Perhaps in the PR discussion page it would be worth mentioning where this flip has been investigated?  Eg. a link to a notebook or some other note explaining why is that change needed.

Reply:
I think part of https://sitcomtn-003.lsst.io/ explains the difference between Zemax and phosim - was it that ts_wep  was following Zemax, and Batoid had the z-axis pointing the opposite way  ?

Reply:
Slide 10 in this presentation from the 2023 PCW shows the comparison between imSim and Phosim: https://docs.google.com/presentation/d/1jMbH0cm74elBpXStOEZ1A40-P4ctHpiXhgngwJtHpmA/edit?usp=sharing 

There is a flip across the y-axis in the donuts so there is a reflection in Zernikes in the x-direction between the two.

There is additional discussion on this issue here in slack cataloging Josh Meyers' exploration of the issue: https://lsstc.slack.com/archives/CHXKSF3HC/p1691456649673369

Reply:
I will note it is still a question under discussion if this should be adjusted here or in OPD so this may change in the future but for current ts_imsim use we have decided to move forward with this solution for the time being.

Reply:
And also Josh Meyers will add a section to [sitcom-tn.](https://sitcomtn-003.lsst.io/) on the batoid coordinate system that will document these differences more formally.